### PR TITLE
Update test case catalog with Best Practices v1.3 references

### PR DESCRIPTION
**Pull request by shimritproj**
Update test case catalog with Best Practices v1.3 references

### CODE REVIEW

File: CATALOG.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Version|v1.0.0
 Description|http://test-network-function.com/testcases/access-control/cluster-role-bindings tests that a Pod does not specify ClusterRoleBindings.
 Result Type|normative
 Suggested Remediation|In most cases, Pod's should not have ClusterRoleBindings.  The suggested remediation is to remove the need for ClusterRoleBindings, if possible.
-Best Practice Reference|[CNF Best Practice V1.2](https://connect.redhat.com/sites/default/files/2021-03/Cloud%20Native%20Network%20Function%20Requirements.pdf) Section 6.2.10 and 6.3.6
+Best Practice Reference|[CNF Best Practice V1.2](https://connect.redhat.com/sites/default/files/2021-03/Cloud%20Native%20Network%20Function%20Requirements.pdf) Section 5.2.10 and 5.3.6
```
Comment (jmontesi) [inline CATALOG.md:21]: Don't you need to change the Best Practices document to point to v1.3?

File: cnf-certification-test/identifiers/identifiers.go [hunk 2]
```diff
@@ -359,7 +359,7 @@ var Catalog = map[claim.Identifier]TestCaseDescription{
 		    In case of deployments that are managed by HPA the test is changing the min and max value to deployment Replica - 1 during scale-in and the 
 			original replicaCount again for both min/max during the scale-out stage. lastly its restoring the original min/max replica of the deployment/s`),
 		Remediation:           `Make sure CNF deployments/replica sets can scale in/out successfully.`,
-		BestPracticeReference: bestPracticeDocV1dot2URL + " Section 6.2",
+		BestPracticeReference: bestPracticeDocV1dot2URL + " Section 5.2",
```
Comment (jmontesi) [inline cnf-certification-test/identifiers/identifiers.go:362]: Same here. I guess this should point to v1.3 of the Best Practices document.

File: CATALOG.md (hunk 1 above)
Comment (shimritproj) [reply CATALOG.md:21]: Agree, I was think that this file update automatic after I was change to point to V1.3. anyway, I changed.

File: CATALOG.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ Version|v1.0.0
 Description|http://test-network-function.com/testcases/access-control/cluster-role-bindings tests that a Pod does not specify ClusterRoleBindings.
 Result Type|normative
 Suggested Remediation|In most cases, Pod's should not have ClusterRoleBindings.  The suggested remediation is to remove the need for ClusterRoleBindings, if possible.
-Best Practice Reference|[CNF Best Practice V1.2](https://connect.redhat.com/sites/default/files/2021-03/Cloud%20Native%20Network%20Function%20Requirements.pdf) Section 6.2.10 and 6.3.6
+Best Practice Reference|https://docs.google.com/document/d/1wRHMk1ZYUSVmgp_4kxvqjVOKwolsZ5hDXjr5MLy-wbg/edit# Section 5.2.10 and 5.3.6
```
[the remark this answers is not in the document]
Comment (shimritproj) [reply CATALOG.md]: This is the link - https://connect.redhat.com/sites/default/files/2022-05/Cloud%20Native%20Network%20Function%20Requirements%201-3.pdf

I changed. @jmontesi

**Review by jmontesi — APPROVED** [review]

**Review by sebrandon1 — APPROVED** [review]

**Pull request merged**